dask_xgboost/core.py

```diff
@@ -315,23 +328,100 @@ def fit(self, X, y=None):
 
         Returns
         -------
-        self : the fitted Regressor
+        self : the fitted XGBRegressor
+
+        Notes
+        -----
+        This differs from the XGBoost version not supporting the ``eval_set``,
+        ``eval_metric``, ``early_stopping_rounds`` and ``verbose`` fit
+        kwargs.
+        """
+        return _xgb_regressor_fit(self, X, y)
+
+    def predict(self, X):
+        return _xgb_regressor_predict(self, X)
+
+
+class XGBRFRegressor(xgb.XGBRFRegressor):
```

Comment:
To unify the two class objects defined here, I think we have a few options:
```
class XGBRegressor(xgb.XGBRegressor, xgb.XGBRFRegressor):
  def __init__(self):
    self.RF = False # boolean flag to determine which regressor to use in Python
  def fit(self, ...):
    if(self.RF): ...
    ...
  def predict(self, ...):
    if(self.RF): ...
    ...

class XGBClassifier(xgb.XGBClassifier, xgb.XGBRFClassifier):
...
```

or something like:
```
class XGBRegressor:
  def __init__(self):
    self.XGBReg = xgb.XGBRegressor()
    self.XGBRFReg = xgb.XGBRFRegressor()
...
```